packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md

```diff
@@ -183,6 +186,17 @@ The OIN Wizard journey includes the **Test integration** experience page to help
 
 > **Note:** You must have the Okta Browser Plugin installed with **Allow in Incognito** enabled before you use the  **OIN Submission Tester**. See [OIN Wizard requirements](/docs/guides/submit-app-prereq/main/#oin-wizard-requirements).
 
+#### Navigate directly to test your integration
+
+You can navigate directly to the OIN Wizard **Test integration** page if you have an existing submission in the **Your OIN Integrations** dashboard. This option is only available if your submission has protocol, configuration, and test information already saved. You can bypass the **[Select protocol](#start-a-submission)**, **[Configure your integration](#configure-your-integration)**, and **[Test your integration](#enter-test-information)** pages in the OIN Wizard, and start generating instances for testing. This saves you time and avoids unnecessary updates to an existing integration submission.
+
+To navigate directly to test your integration:
+
+1. From the **Your OIN Integrations** dashboard, select the more menu icon (![three-dot more icon](/img/icons/odyssey/more.svg)) next to the integration submission you want to test.
```

Comment:
Should more be capitalized and in bold?

Reply:
"more" isn't in the UI. The icon is the three-dot "action menu". When I asked the infodev-standards channel earlier for this, they recommended that I use "more" to describe the icon. I'm open to using another term for this icon.